DeadmansSwitch.sol

```diff
@@ -4,33 +4,49 @@ pragma solidity ^0.8.0;
 contract DeadmansSwitch {
     // TODO: Declare state variables
     // Hint: You'll need variables for the owner, beneficiary, and last check-in block
+    address owner;
+    address beneficiary;
+    uint check_in_block;
 
     // TODO: Implement constructor
     constructor(address _beneficiary) {
         // Hint: Initialize state variables
+        owner=msg.sender;
+        beneficiary=_beneficiary;
     }
 
     // TODO: Implement still_alive function
     function still_alive() public {
         // Hint: Update the last check-in block
+        check_in_block=block.number;
     }
 
     // TODO: Implement release funds function
-    function releaseFunds() public {
+    function releaseFunds() public payable{
```

Comment:
Also, there is no transfer of balance in case the 10th block has passed since the last check-in